flask/app/variants.py

```diff
@@ -185,7 +186,12 @@ def summary(type: str):
 
     ensgs = parse_gene_panel()
 
-    # app.logger.debug(ensgs)
+    alias_subquery = aliased(
```

Comment:
Cool solution to an ensembl id not always having an HGNC gene symbol! I think it'd be helpful if the rationale for this approach was explained, maybe in the docstrings or a comment here